source/Calamari/AzureContextScriptWrapper.cs

```diff
@@ -68,14 +68,14 @@ public AzureContextScriptWrapper(IVariables variables, ICalamariFileSystem fileS
                     SetOutputVariable("OctopusAzureADTenantId", variables.Get(SpecialVariables.Action.Azure.TenantId)!);
                     SetOutputVariable("OctopusAzureADClientId", variables.Get(SpecialVariables.Action.Azure.ClientId)!);
                     variables.Set("OctopusAzureADPassword", variables.Get(SpecialVariables.Action.Azure.Password));
-                    return NextWrapper.ExecuteScript(new Script(contextScriptFile.FilePath), scriptSyntax, commandLineRunner, environmentVars);
+                    return NextWrapper!.ExecuteScript(new Script(contextScriptFile.FilePath), scriptSyntax, commandLineRunner, environmentVars);
```

Comment:
A bit dodgy, but we know it is not the last one, the framework ensures that

Reply:
Why did we need to make this change?

Reply:
Becasue `NextWrapper` can now be nullable